src/v2/Apps/Auctions/Components/StandoutLots/StandoutLots.tsx

```diff
@@ -22,7 +22,20 @@ const StandoutLots: React.FC<StandoutLotsProps> = ({ viewer }) => {
   const nodes = extractNodes(viewer.standoutLotsConnection)
 
   if (nodes.length === 0) {
-    return null
+    return (
+      <Box>
+        <Text
+          as="h3"
+          color="black60"
+          mb={12}
+          mt={6}
```

Comment:
Do these margins work in the mobile view?

Reply:
They do, updated with Mobile screen shot above.